python/cugraph/cugraph/dask/sampling/uniform_neighbor_sample.py

```diff
@@ -612,6 +614,16 @@ def uniform_neighbor_sample(
         fanout_vals = fanout_vals.get().astype("int32")
     elif isinstance(fanout_vals, cudf.Series):
         fanout_vals = fanout_vals.values_host.astype("int32")
+    elif isinstance(fanout_vals, dict):
+        # FIXME: Add expensive check to ensure all dict values are lists
+        # Convert to a tuple of sequence (edge type size and fanout values)
+        edge_type_size = []
+        [edge_type_size.append(len(s)) for s in list(fanout_vals.values())]
```

Comment:
Does this iterate over the edge types in the dictionary in order?  We need to make sure that this is constructed with edge type 0 first, followed by edge type 1, etc.

Reply:
Right. I converted the heterogeneous fanout type to a sorted ordered dictionary.